pom.xml

```diff
@@ -81,6 +81,9 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
 
+        <otel.alpha.version>1.19.0-alpha</otel.alpha.version>
```

Comment:
1.19.2 is out

Reply:
Thanks! Actually I don't think I need that variable ( used it when testing the autoconfigurespi)